ModernFlyouts/FlyoutWindow.xaml.cs

```diff
@@ -213,15 +213,15 @@ private void OtherPreps()
 
         internal void SetUpAnimPreps()
         {
-            T1.Y = -40;
-            ContentGrid.Opacity = 0;
+            T1.SetCurrentValue(System.Windows.Media.TranslateTransform.YProperty, (double)-40);
+            if (ContentGrid != null) ContentGrid.SetCurrentValue(OpacityProperty, (double)0);
```

Comment:
Please revert every change that introduced `SetCurrentValue`. This is insane! What happened to Intellisense!
Is it on drugs?